content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -100,6 +104,42 @@ On your validator:
 
 
 
+## Connect to the Network Using Public Hubs
+
+This section applies to running a validator that you want to remain an untrusted validator. In this case, it may be more efficient for you to use public hubs to connect your validator to the network than to run stock `rippled` servers to use as proxies.
+
+**Note:** If you are running a trusted validator or a validator that you want to become trusted, [connect your validator to the network using proxies](#connect-to-the-network-using-proxies) instead of public hubs.
+
+As a validator operator, one of your key responsibilities is to ensure that your validator has reliable and safe connections to the XRP Ledger network. Instead of connecting to random and potentially malicious peers on the network, you can instruct your validator to connect to the network through one or more public hubs. This setup can help protect your validator from [DDoS](https://en.wikipedia.org/wiki/Denial-of-service_attack) attacks.
+
+A public hub is a `rippled` server that Ripple has assessed to have a track record of embodying the following traits: ***TODO: accurate to say that Ripple assessed this?***
+
+- Good bandwidth.
+- Connections to a lot of peers with good reputations. ***TODO: I imagine that the assessment included checking the behavior of the peers? Is it accurate to refer to this as "reputations"?***
+- Ability to relay messages reliably.
+
+To connect your validator reliably and safely to the network using public hubs, set the following configurations in your validator’s `rippled.cfg` file.
+
+1. Include the following `[ips_fixed]` stanza. The two values, `r.ripple.com 51235` and `zaphod.alloy.ee 51235`, are public hubs. This stanza tells `rippled` to always attempt to maintain peer connections with these public hubs.
+
+        [ips_fixed]
+        r.ripple.com 51235
+        zaphod.alloy.ee 51235
+
+    You can include the IP addresses of other `rippled` servers you trust, but _**do not include**_ servers that you don’t trust.
```

Comment:
It might be good to indicate what you are supposed to trust them to do / not to do. I believe this is a much lower level of trust than "has the keys to my house" but a higher level of trust than "person who approached me in the BART station today."

It's also distinct from validator trust.

I _think_ the answer is that these are hubs you expect not to go offline, DDoS you, or try to crash you. You expect them to relay messages more or less unconditionally without censoring. And you expect them not to publish your IP address to strangers.